lib/rubocop/cop/minitest/global_expectations.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Minitest
+      # This Cop checks for deprecated global expectations
+      # and autocorrects them to use expect format.
+      #
+      # @example
+      #   # bad
+      #   n.must_equal 42
+      #   n.wont_match b
+      #
+      #   # good
+      #   _(n).must_equal 42
+      #   _(n).wont_match b
+      class GlobalExpectations < Cop
+        MSG = 'Prefer using `%<corrected>s`.'
+
+        VALUE_MATCHERS = %i[
+          be be_close_to be_empty be_instance_of be_kind_of
+          be_nil be_same_as be_silent be_within_epsilon equal
+          include match respond_to must_exist
+        ].map do |matcher|
+          [:"must_#{matcher}", :"wont_#{matcher}"]
+        end.flatten.freeze
+
+        BLOCK_MATCHERS = %i[must_output must_raise must_throw].freeze
+
+        MATCHERS_STR = (VALUE_MATCHERS + BLOCK_MATCHERS).map do |m|
+          ":#{m}"
+        end.join(' ').freeze
+
+        def_node_matcher :global_matcher?, <<~PATTERN
```

Comment:
`:global_expectations?` may be more suitable.

```suggestion
        def_node_matcher :global_expectations?, <<~PATTERN
```

Reply:
@koic Changed.

Reply:
I went ahead & also changed the constant names from `MATCHERS` to `EXPECTATIONS` for consistency.

Reply:
I think `MATCHERS` is fine for constant names because `MATCHERS` is part of `def_node_matcher :global_expectations?` :-)

Reply:
Alright reverted back :)